cfep-01.md

```diff
@@ -0,0 +1,135 @@
+of CFEPs in a civil and respectful manner.  The CFEP champion(s) should be
+prepared to answer questions on the proposal and make updates to the
+proposal as recommended by the community.
+
+## Resolution
+
+All CFEPs will be resolved as either *Rejected*, *Accepted* or *Deferred*
+depending on the consensus of the community. Once the community has reached a
+consensus, the CFEP champion should update the Status of proposal and add a
+link to the discussion to the table.  Regardless of the resolution, the pull
+request should be merged once these tasks have been done.
+
+## CFEP Maintenance
+
+For *Rejected* CFEPs, no modifications are made after the pull request is merged.
+*Accepted* CFEPs should be updated with a link to the pull request(s) of the
+implementation and the status changed to *Implemented* when the proposed
+changes have been implemented.  *Deferred* CFEPs can be re-submitted in a new
+pull request after an appropriate amount of time.
+
+# CFEP Content
+
+All CFEPs should begin with a top level table with the following information:
+
+<table>
+<tr><td> Title </td><td> A short title of the proposal </td>
+<tr><td> Status </td><td> Draft | Proposed | Accepted | Rejected | Deferred | Implemented </td></tr>
+<tr><td> Author(s) </td><td> Full Name &lt;full.name@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Aug 30, 2016</td></tr>
+<tr><td> Updated </td><td> Aug 30, 2016</td></tr>
+<tr><td> Discussion </td><td> link to the issue where the CFEP is being discussed, NA before submission </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not available </td></tr>
+</table>
+
+This table should be followed by a **Abstract** section and then
+additional sections as needed by the proposal.  Some section that may be
+included if appropriate for the proposal include.
+
+    * Specification -- The technical details of the proposed change.
+    * Motivation -- Why the proposed change is needed.
+    * Rationale -- Why particular decisions were made in the proposal.
+    * Backwards Compatibility -- Will the proposed change break existing
+      packages or workflows.
+    * Alternatives -- Any alternatives considered during the design.
+    * Sample Implementation -- Links to prototype or a sample implementation of
+      the proposed change.
+    * FAQ -- Frequently asked questions (and answers to them).
+    * Reference -- Any references used in the design of the CFEP.
+
+A final **Copyright** section is also required.
+
+## References
+
+Much of this document was adapted from
+[PEP 1 -- PEP Purpose and Guidelines](https://www.python.org/dev/peps/pep-0001/)which has been placed in the public domain.
+The [IPEPs: IPython Enhancement Proposals](https://github.com/ipython/ipython/wiki/IPEPs:-IPython-Enhancement-Proposals) was also referenced for inspiration.
+
+## Copyright
+
+This document has been placed in the public domain.
```

Comment:
Yeah I think someone mentioned this elsewhere, but Public Domain doesn't exist everywhere so CC-0 seems like a sensible choice that avoids these problems. That all said IANAL.